docs/PersonalApplications.rst

```diff
@@ -504,6 +510,103 @@ your own Wiki page. See the :ref:`Working With Wikis <Wiki>` chapter for the ful
         However, if you have :ref:`made public <MakingPublic>` for your own wiki, anyone can access, read and edit it via the link.
 
 
+.. _Managing-Fav-Apps:
+
+=======================================
+Managing your favorite applications
+=======================================
+
+Starting with eXo Platform 6, you can quickly access your favourite applications 
```

Comment:
favourite or favorite ? 
sometimes you use favourite, sometimes you use favorite. It should be consistent

Reply:
Ok i'll change to favorite